enhancements/service-binding.md

```diff
@@ -0,0 +1,291 @@
+
+
+
+## Motivation
+
+
+Connecting applications to the services that support them—for example, establishing the exchange of credentials between a Java application and a database that it requires—is referred to as binding. The configuration and maintenance of this binding together of applications and backing services can be a tedious and inefficient process. Manually editing YAML files to define binding information is error-prone and can introduce difficult-to-debug failures.
+
+The goal of the Service Binding Operator is to solve this binding problem. By making it easier for application developers to bind applications with needed backing services, the Service Binding Operator also assists operator providers in promoting and expanding the adoption of their operators
+
+
+## The ServiceBinding custom resource
+
+Manually binding an application together with a backing service without the Service Binding Operator is a time-consuming and error-prone process. 
+
+The steps needed to perform the binding include:
+
+* Locating the binding information in the backing service’s resources.
+* Creating and referencing any necessary secrets.
+* Modifying the application’s DeploymentConfig, Deployment, Replicaset, KnativeService, or anything else that uses a standard PodSpec to reference the binding request.
+
+In contrast, by using the Service Binding Operator, the only action that an application developer must make during the import of the application is to make clear the intent that the binding must be performed. This task is accomplished by creating the `ServiceBinding`. The Service Binding controller takes that intent and performs the binding on behalf of the application developer.
+
+The Service Binding Operator accomplishes this task by automatically collecting binding information and sharing it with an application and an operator-managed backing service. This binding is performed through a new custom resource called a `ServiceBinding`.
+
+
+```yaml
+apiVersion: service.binding/v1alpha1
+kind: ServiceBinding
+
+metadata:
+  name: binding-ecommerce
+  namespace: foo
+
+spec:
+
+  applications:
+  - name: payments-app
+    group: ""
+    version: v1
+    resource: deployments
+
+  services:
+  - group: postgresql.baiju.dev
+    version: v1alpha1
+    kind: Database
+    name: db-demo
+
+```
+
+## Adding binding guidance to Services
+
+
+The Service Binding machinery respects binding-related guidance which backing service operators choose to provide. 
+
+
+### Backing Service providing binding metadata
+
+
+If the backing service author has provided binding metadata in the corresponding CRD,
+then Service Binding acknowledges it and automatically creates a binding secret with
+the information relevant for binding.
+
+The backing service may provide binding information as
+* Metadata in the CRD as annotations
+* Metadata in the OLM bunde manifest file as Descriptors
```

Comment:
is this referring to the CSV?  (also typo on bundle)

Reply:
Yes!